honeybee_radiance/command/options/rcontrib.py

```diff
@@ -0,0 +1,234 @@
+from .optionbase import BoolOption, StringOption, StringOptionJoined, IntegerOption, \
+    FileOption
+from .rtrace import RtraceOptions
+import honeybee_radiance.exception as exception
+import warnings
+
+
+# TODO: Add input pattern check for -p
+class RcontribOptions(RtraceOptions):
+    """rcontrib command options.
+
+    [ -n nprocs ][ -V ][ -c count ][ -fo | -r ][ -e expr ][ -f source ][ -o ospec ]
+    [ -p p1=V1,p2=V2][ -b binv ][ -bn nbins ] { -m mod | -M file } [ $EVAR ] [ @file ]
+    [ rtrace options ]
+    
+    NOTE:
```

Comment:
Again, I am not sure if this is supposed to be all caps.

Reply:
Also, I see in some places you use `NOTE:` and in others you use `Also see:`  You might just want to be consistent about this.